pkg/oam/mock/mocks.go

```diff
@@ -30,7 +30,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 
-	"github.com/crossplane/crossplane-runtime/apis/common/v1"
+	v1alpha1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
```

Comment:
This isn't alpha1, right?

Reply:
Yes, it's an alias to reduce the code changes from the original fork.